ng/src/gmp/http/transform/x2js.js

```diff
@@ -72,6 +72,10 @@ class X2JsTransform extends Transform {
         }
 
         if (is_defined(envelope.action_result)) {
+          if (is_defined (envelope.action_result.details)
+              && envelope.action_result.details !== '') {
```

Comment:
Can this ever happen? I don't like tests for empty string. Either it should not defined or it should contain useful information.